Automattic/wp-calypso
# Stats: Fix emoji thresholds in PWYW flow

Related to #85352.

## Proposed Changes

Uses index values as computed by the caller. These values are based on the total number of steps which are currency dependant and they are determined as a percentage of the maximum PWYW tier.

## Testing Instructions



* Launch the Calypso Live link.
* Navigate to **Stats → Traffic**.
* Click on CTA to purchase Stats plan.
* Add `stats/tier-upgrade-slider` to URL flags.
* Confirm the new tier slider shows the ❤️ emoji for the default tier (the middle tier).
* Confirm the slider shows the 🔥 emoji for the top tier (top 80% of available tiers).
* Confirm no emoji or "Thank you!" message for the lowest tier.

<img width="581" alt="SCR-20231215-qhpk" src="https://github.com/Automattic/wp-calypso/assets/40267301/7bc845f4-9450-4d41-94c5-a6f2898aa9b4">

## Pre-merge Checklist
